### DISCUSSION TITLE
Education Application made difficult by no image upload

### DISCUSSION THREAD

**Discussion by sabinevidal** [community/community#131787] [GitHub Education]
### Hi everyone,

Hi there,
I have tried to submit my application multiple times but it seems to not accept it as I never receive a successfully applied (or unsuccessful) notification. I've tried taking multiple photos of my documentation, but only being able to use my webcam or front selfie phone camera (iphone 15) is making it difficult to get good quality images. I'm being asked to prove why I'm not on campus and the only information I have about this is over emails and on the university webpage (I'm a remote student) so I'm having to take a photo with my (bad) webcam of my phone which means the quality is bad. 

Please allow for image uploads again.

**Comment by paiineli** [comment]
Hi,

I understand the frustration you're facing with your application. It sounds like the current photo submission method is making it difficult to provide clear documentation.

It might help to reach out directly to GitHub Support and explain your situation, emphasizing your status as a remote student and the challenges with using a webcam for quality images. Requesting the ability to upload images directly may be beneficial, as it could simplify the process for you and others in similar situations.

I hope you find a resolution soon!

Best,

**Comment by samus-aran** [comment]
Hi @sabinevidal 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.


- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

> We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.


1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
2. If you can, print off your evidence and take a photo of this copy.
3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
5. Alternatively you may use another mobile device or computer.

**Comment by MillerAnthony-FS** [comment]
I think it is a terrible system. I finally went to my local library branch and had a document showing my enrollment and a Google map showing I live a state away from the university printed and I used those to take pictures. That worked.
